### PR TITLE
fix: print correct tag

### PR DESCRIPTION
**Pull request by KyleTryon**

### CODE REVIEW

File: src/scripts/publish.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ function publishDevOrbs() {
   {
     printf "Your development orb(s) have been published. It will expire in 30 days.\n"
     printf "You can preview what this will look like on the CircleCI Orb Registry at the following link: \n"
-    printf "https://circleci.com/developer/orbs/orb/%s?version=dev:%s\n" "${ORB_VAL_ORB_NAME}" "${array[0]}"
+    printf "https://circleci.com/developer/orbs/orb/%s?version=%s\n" "${ORB_VAL_ORB_NAME}" "${array[0]}"
```
[the remark this answers is not in the document]
Comment (KyleTryon) [reply src/scripts/publish.sh:48]: Slowly working towards it. Now i just ran into an unhandled authentication error, so we are going to squash  few things here lol

**Comment by wyardley** [comment]
Fixes #204

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:${CIRCLE_SHA1}

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:${CIRCLE_SHA1}

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:${CIRCLE_SHA1}

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:${CIRCLE_SHA1}

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:${CIRCLE_SHA1}

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:${CIRCLE_SHA1}

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:${CIRCLE_SHA1}

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
 printf 'https://circleci.com/developer/orbs/orb/%s?version=%s
' circleci/orb-tools dev:a83d7363580b4672ef65e0e1d644e8a74fda6a0a printf 'https://circleci.com/developer/orbs/orb/%s?version=%s
' circleci/orb-tools dev:alpha

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
 https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:4116f3979421a1b4c5b21a97475908ccd482ba54 https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:alpha

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:22bac157c86558f5d48be636b956d3b3b1554fe3https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:alpha

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:b544b2bd892db174edb625de0646860cb736a670
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:alpha

**Review by EricRibeiro — APPROVED** [review]

**Review by EricRibeiro — APPROVED** [review]

**Review by EricRibeiro — APPROVED** [review]

File: src/scripts/publish.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (KyleTryon) [reply src/scripts/publish.sh]: we got it 🎉

**Comment by KyleTryon** [comment]
image

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:56043f6d87214a3037605feb96700e2cc5b37c7a
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:alpha

**Comment by wyardley** [comment]
> Your development orb(s) have been published. It will expire in 30 days.
> You can preview what this will look like on the CircleCI Orb Registry at the following link(s):

Will note that tense is a little inconsistent there. Not sure if it should stick with plural or singular throughout, but maybe pick one or the other? (it vs they etc.)

File: src/scripts/publish.sh [hunk 2]
```diff
@@ -37,18 +37,23 @@ function publishOrb() {
...
 function publishDevOrbs() {
   printf "Publishing development orb(s).\n\n"
+  ORB_REG_LINKS=()
   DEV_TAG_LIST=$(echo "${ORB_VAL_DEV_TAGS}" | tr -d ' ')
   IFS=',' read -ra array <<<"$DEV_TAG_LIST"
   for tag in "${array[@]}"; do
-    publishOrb "$tag"
+    # shellcheck disable=SC2005
+    PROCESSED_TAG=$(circleci env subst "$tag")
+    ORB_REG_LINKS+=("$(printf 'https://circleci.com/developer/orbs/orb/%s?version=%s' "$ORB_VAL_ORB_NAME" "$PROCESSED_TAG")")
```
Comment (wyardley) [inline src/scripts/publish.sh:46]: 🔥

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:2dd20705b2331ef59dce39abb4e8310dea484f92
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:alpha

**Comment by orb-publisher** [comment]
Your development orb(s) have been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link(s): 
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:2dd20705b2331ef59dce39abb4e8310dea484f92
https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=dev:alpha

**Pull request merged**